pytube/extract.py

```diff
@@ -278,7 +278,11 @@ def apply_descrambler(stream_data: Dict, key: str) -> None:
     if key == "url_encoded_fmt_stream_map" and not stream_data.get(
         "url_encoded_fmt_stream_map"
     ):
-        formats = json.loads(stream_data["player_response"])["streamingData"]["formats"]
+        streamingData = json.loads(stream_data["player_response"])["streamingData"]
+        if 'formats' in streamingData.keys():
+            formats = json.loads(stream_data["player_response"])["streamingData"]["formats"]
+        elif 'adaptiveFormats' in streamingData.keys():
+            formats = json.loads(stream_data["player_response"])["streamingData"]["adaptiveFormats"]
         formats.extend(
             json.loads(stream_data["player_response"])["streamingData"][
                 "adaptiveFormats"
```

Comment:
This should be removed I guess. Since `adaptiveFormats` already comes from the above `elif` statement and this would repeat the stream formats.

Reply:
Removing this should also fix the above traceback.